### DISCUSSION TITLE
Updgrading to Next.js 9.4.2 caused annoying error popups in local development

### DISCUSSION THREAD

**Discussion by olpeh** [vercel/next.js#13661] [Help]
After upgrading, we started seeing some kind of error popups in dev mode. Is this a Next.js feature? If so, how can we *disable* this? This makes no sense to me

![image](https://user-images.githubusercontent.com/6113341/83516582-0747d880-a4e0-11ea-86a8-5d1619ff1a86.png)

**Comment by timneutkens** [comment]
Can you provide a reproduction? Sounds like you have uncaught runtime errors

**Reply by olpeh** [reply]
No, not really, since this does not seem to be a bug. Well, yeah we are having an uncaught error, that's not what I was asking about. 

How do we get rid of these popups?

**Reply by timneutkens** [reply]
As of right now you can't disable the error overlay given that it's a runtime error. Hence why I'm asking for a reproduction so that we can evaluate if it's a valid case to allow disabling it. Runtime errors are bugs right 🤔 We can also improve the stacktrace (doesn't seem to show one now) if we have a reproduction.

**Reply by olpeh** [reply]
Okay, I'll need to try reproducing something. Yeah maybe the stack trace would actually make this a somewhat useful feature to have.

**Reply by olpeh** [reply]
I was not able to reproduce this with a simple setup and unfortunately can not share our codebase. The issue probably is that we have something that fails locally, but not in any deployed environments and that's why we haven't fixed this. It's a large codebase, so I'm not able to figure out what fails due to the missing error stack trace.

**Comment by danilockthar** [comment]
The previous error overlay helps more than the new one :/, there is a way to go back to the previous error overlay ?

**Reply by timneutkens** [reply]
Please provide a complete reproduction of what you think was better so that we can improve the new overlay.
